public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+        font-family: GothamBold, Arial, Helvetica, sans-serif;
+    }
+    .survey-row-expansion {
+        border-left: #cccccc 1px solid;
+        border-right: #cccccc 1px solid;
+    }
+    .sub-table-1 .sub-question-text, .sub-table-2 .sub-question-text {
+        padding-top: 1rem;
+    }
+    .survey-sub-tbl {
+        border-bottom: #cccccc 1px solid;
+    }
+    .survey-sub-tbl .survey-tbl-exp-r:last-of-type {
+        border-bottom: none;
+    }
+
+    .survey-sub-table {
+        margin-left: 2%;
+        width: 98%;
+    }
+
+    .survey-sub-table-2 {
+        padding-top: 1rem;
+        margin-left: 2%;
+        width: 98%;
+    }`;
+
+
+interface SurveyRowProps {
+    // survey_concept_id
+    stratum1: number;
+    // question_concept_id
+    stratum2: number;
+    // answer_concept_id
+    stratum3: number;
+    // answer_value_string
+    stratum4: string;
+    // hasSubQuestions
+    stratum7: string;
+    countValue: number;
+    drawerOpen: boolean;
+    countPercent: number;
+    isCopeSurvey: boolean;
+    partcipantCount: number;
+}
+
+interface SurveyRowState {
+    drawerOpen: boolean;
+    subQuestion: object;
+    subAnswers: object;
+    subTitle: string;
+}
+
+
+export const SurveyAnswerRowComponent = (class extends React.Component<SurveyRowProps, SurveyRowState> {
+    constructor(props: SurveyRowProps, state: SurveyRowState) {
+        super(props);
+
+        this.state = {
+            drawerOpen: false,
+            subQuestion: {},
+            subAnswers: {},
+            subTitle: ''
+        };
+    }
+    surveyConceptId = this.props.stratum1;
+    hasSubQuestions = this.props.stratum7;
+
+
+    openDrawer(e) {
+        this.setState({
+            drawerOpen: !this.state.drawerOpen
+        });
+        if (this.hasSubQuestions === '1') {
+            this.getSubQuestions(1);
+        }
+    }
+    getSubQuestions(level: number) {
+        return api.getSubQuestions(this.surveyConceptId, this.props.stratum2, this.props.stratum3, 1)
+            .then(
+                results => {
+                    console.log(results);
+
+                    // this.props.SubQuestion.countAnalysis = results.items.filter(a => a.analysisId === 3110)[0];
+                    // console.log(results.questions.items[0].countAnalysis.results);
+                    this.setState({
+                        subQuestion: results.questions.items[0],
+                        subAnswers: results.questions.items[0].countAnalysis.results,
+                        subTitle: results.questions.items[0].conceptName
+                    });
+                    console.log(this.state.subAnswers, 'subAnswers');
+                    // console.log(this.state.subQuestion, 'Sub question');
+
+                });
+    }
+
+    render() {
+        const parcipantPercentage = this.props.countValue / this.props.partcipantCount;
+        return <React.Fragment> <div className='survey-tbl-exp-r survey-tbl-r' onClick={this.openDrawer.bind(this)}>
```

Comment:
Can use an inline function instead of binding here:
```suggestion
        return <React.Fragment> <div className='survey-tbl-exp-r survey-tbl-r' onClick={() => this.openDrawer()}>
```